cogs/events/error.py

```diff
@@ -0,0 +1,120 @@
+            return await ctx.send(f"{config.crossmark} **I'm missing {perms} permissions**")
+
+        if isinstance(err, commands.MissingRequiredArgument):
+            return await ctx.send(f"**You are missing required arguments - {err.param}**")
+
+        if isinstance(err, commands.CommandOnCooldown):
+            clog = await self.bot.fetch_channel(837690526789533697)
+            e = discord.Embed(color=discord.Color.red())
+            e.description = f"**{ctx.author} has a `{err.retry_after:.0f}` second cooldown on command `{ctx.command.qualified_name}`**" \
+                            f"\nGuild **{ctx.guild}** with ID `{ctx.guild.id}` | User ID: `{ctx.author.id}`"
+            await clog.send(embed=e)
+            return await ctx.send(f"`{ctx.command.qualified_name}` **is on cooldown for __{err.retry_after:.0f}__ more seconds.**")
+
+        if isinstance(err, commands.NotOwner):
+            return await ctx.send(f"{config.crossmark} **Only __bot owners__ can use this command.**")
+
+        if isinstance(err, commands.MemberNotFound):
+            return await ctx.send(f"{config.confused} **Could not find user `{err.argument}`**")
+
+        if isinstance(err, commands.ChannelNotFound):
+            return await ctx.send(f"{config.confused} **Could not find channel `{err.argument}`**")
+
+        if isinstance(err, commands.MessageNotFound):
+            return await ctx.send(f"{config.confused} **Could not find message `{err.argument}`**")
+
+        if isinstance(err, commands.RoleNotFound):
+            return await ctx.send(f"{config.confused} **Could not find this role**")
+
+        if isinstance(err, commands.NoPrivateMessage):
+            return await ctx.send(f"{config.crossmark} **You can only use this command in servers.**")
+
+        if isinstance(err, commands.DisabledCommand):
+            return await ctx.send(f"{config.crossmark} **{ctx.command.qualified_name} is currently disabled.**")
+
+        if isinstance(err, commands.CheckFailure):
+            await ctx.send(f"{config.crossmark} **You do not have permission to use this command.**")
+
+        else:
+
+            elog = await self.bot.fetch_channel(838000482894610462)
+            le = discord.Embed(color=discord.Color.red())
+            le.description = f"__**Full traceback**__" \
+                            f"\n```py\n{''.join(traceback.format_exception(type(err), err, err.__traceback__))}\n```"
+            le.set_author(name=f"{ctx.author} | {ctx.author.id} (Guild {ctx.guild.id})", icon_url=ctx.author.avatar_url)
+            await elog.send(embed=le)
+
+            def check(r, u):
+                return u.id == ctx.author.id and r.message.id == checkmsg.id
+
+            e = discord.Embed(title="traceback", color=discord.Color.red())
+            e.description = f"```py\n{''.join(traceback.format_exception(type(err), err, err.__traceback__))}\n```"
+            e.set_footer(text="Do you want a developer to join and investigate?")
```

Comment:
I'd add `if ctx.channel.permissions_for(ctx.guild.me).create_invite` and update the code to not prompt for reaction_add if the bot is missing permissions, aka just send an embed with "Error has been reported to our developer(s)" when the bot is missing permissions and if it's not missing permissions to create an invite, ask if they would like to invite one of the developers to investigate.